certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
```

Comment:
That's fair. It was just a thought to add some extra protections around the non-interactive case. We could set `n` high enough where we don't expect a user to ever hit it, but I don't expect this to be a problem.